easybuild/easyconfigs/g/git/git-2.2.2-GCC-4.9.2.eb

```diff
@@ -0,0 +1,46 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# Authors::   Dmitri Gribenko <gribozavr@gmail.com>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-90.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'git'
+version = '2.2.2'
+
+homepage = 'http://git-scm.com/'
+description = """Git is a free and open source distributed version control system designed
+to handle everything from small to very large projects with speed and efficiency."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+# eg. http://git-core.googlecode.com/files/git-1.8.2.tar.gz
+sources = ['v%s.tar.gz' % version]
+source_urls = ['https://github.com/git/git/archive']
+
+preconfigopts = 'make configure && '
```

Comment:
so the correct sequency is `make configure && ./configure && make && make install`?

oh man, the madness, if even git can't get this right... >_<